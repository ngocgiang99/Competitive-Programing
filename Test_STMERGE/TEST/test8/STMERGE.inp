8
2 3
122469817 930633333 851180609 
311490554 316103235 818786007 
4 4
811726579 299914338 547621674 739367293 
816793326 575195594 324524204 620648012 
155525528 736496646 20710952 711590529 
113239191 360347209 785510243 365265264 
1 1
119637090 
8 10
761986811 61954922 19620881 463332289 340706580 814667521 843055021 354041220 54102153 385931754 
902864344 703011733 613462479 961221420 727836966 725294361 76108186 599706635 490265196 802100084 
68608225 697242124 209189056 572916812 55643863 590922484 566534007 62375302 288894707 31994057 
740855844 399275334 687887592 927973326 900606622 529660402 250968748 499853450 996126970 765217327 
346693748 964194240 600015755 7223521 564513973 45227639 464002281 442331950 944674851 284759027 
324090342 132933754 181190581 419926969 216560537 156157547 93756063 249331599 260400982 101911270 
162874453 276638584 644007603 175588879 375321229 874082747 723911556 428983884 604127037 608292571 
909074911 870888002 948591768 97255327 206143000 368278229 352323860 252903691 88853188 770413867 
4 6
537742009 939393705 101538434 560689968 22773003 868219123 
417720223 964114226 620162319 616414191 620723761 376093297 
527553115 305674689 434711020 293304624 504106179 74606800 
642026694 831287798 968492637 602767294 420130549 977942407 
5 9
302599404 457698536 733193913 289867538 514921825 937249118 549031140 749293569 147794761 
533682287 162067734 745676260 89889752 317969374 828936353 109913761 682336901 913322418 
413627164 931626588 482248964 375548463 39050844 745953660 426559305 509346778 228062836 
741442212 823365514 890971000 942508988 941537465 996902973 884085774 398029510 346327777 
63597077 508616418 773674601 586137217 59982344 391417207 849110969 836311358 744439115 
4 10
257674492 533370706 144115004 636542198 378047969 896221652 344891574 961041485 701352017 409884357 
607642759 702323237 271337202 301792345 944548210 214709971 259196073 827119367 774223485 361615085 
240316058 997908354 455223534 832991559 679325739 931540672 931045960 235738358 250604800 675462134 
125573202 671157584 33111296 418291865 780805392 674351081 352080462 760112222 264824007 4316823 
5 3
852173820 434869102 621949097 
440418657 890284405 266450627 
142292801 757712714 605851701 
10642732 159006779 709707688 
309978499 654278013 118737150 
