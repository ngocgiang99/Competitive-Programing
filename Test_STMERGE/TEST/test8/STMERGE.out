311490554
113239191
119637090
116876208
473473100
63597077
125573202
129379882
